src/components/StorageOverview/OCSHealth/HealthBody.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
We should extract some parts of Health to `Dashboard` but I think this is fine for now, we can do it as a follow up